src/app/child-dev-project/children/model/child.ts

```diff
@@ -135,19 +135,4 @@ export class Child extends Entity {
       super.isActive
     );
   }
-
-  /**
-   * @override see {@link Entity}
-   */
-  @DatabaseField() get searchIndices(): string[] {
-    let indices = [];
-
-    indices = indices.concat(this.toString().split(" "));
-    if (this.projectNumber !== undefined && this.projectNumber !== null) {
```

Comment:
let's add `searchable` to the projectNumber schema to replicate existing behavior